[2019-09-11 15:20:51,759] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 15:20:51,761] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 15:20:52,110] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 15:20:52,247] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 15:21:11,840] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 15:21:12,189] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 15:22:08,504] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 15:22:10,903] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 15:29:20,505] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 15:31:22,092] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 15:31:55,870] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 15:32:54,599] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 15:33:36,835] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 15:33:52,971] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 15:35:52,238] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 15:36:13,256] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 15:36:34,341] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 15:36:52,527] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 15:41:05,363] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 15:47:04,803] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 15:47:42,632] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 15:48:36,134] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 15:54:02,658] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 15:57:54,753] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 15:58:26,061] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 15:58:47,131] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 15:59:22,926] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 15:59:35,226] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 16:00:11,797] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 16:03:08,084] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 16:07:00,730] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 16:11:26,803] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 16:12:23,135] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 16:12:40,755] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 16:13:17,609] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 16:14:40,172] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 16:15:08,763] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 16:15:38,760] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 16:16:19,722] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 16:18:26,894] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 16:18:56,037] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 16:20:00,019] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 16:20:47,552] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 16:21:21,862] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 16:21:46,023] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 16:23:04,406] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 16:23:39,812] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 16:47:55,196] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 16:49:41,459] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 16:51:02,154] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 16:52:24,796] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 17:00:48,468] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 17:04:51,985] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 17:08:37,188] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 17:11:09,824] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 17:13:24,804] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 17:15:19,591] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 17:16:15,132] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 17:16:32,469] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 17:16:51,075] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 17:17:08,748] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 17:18:28,051] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 17:18:53,360] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 17:23:35,413] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 17:30:17,567] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 17:31:53,099] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 17:32:04,340] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 17:32:05,663] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 17:35:20,112] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 17:35:21,034] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 17:37:13,225] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 17:37:13,859] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 17:37:30,657] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 17:37:30,996] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 17:39:02,116] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 17:39:02,930] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 17:47:31,597] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 17:47:34,090] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 17:55:53,856] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 17:57:08,767] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 18:07:15,925] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 18:07:16,309] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 18:11:38,539] {add_segment_master.py:42} ERROR - Exception occured
Traceback (most recent call last):
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1249, in _execute_context
    cursor, statement, parameters, context
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/engine/default.py", line 552, in do_execute
    cursor.execute(statement, parameters)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/MySQLdb/cursors.py", line 209, in execute
    res = self._query(query)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/MySQLdb/cursors.py", line 315, in _query
    db.query(q)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/MySQLdb/connections.py", line 226, in query
    _mysql.connection.query(self, query)
MySQLdb._exceptions.OperationalError: (1054, "Unknown column 'nsr_table_information.Qty_Apr' in 'field list'")

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/ganesh/my_projects/itc_poc_server/controllers/segment_master/add_segment_master.py", line 35, in get
    data = data_schema.dump(categories)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow/schema.py", line 539, in dump
    result = self._serialize(processed_obj, many=many)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow/schema.py", line 500, in _serialize
    return [self._serialize(d, many=False) for d in obj]
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow/schema.py", line 500, in <listcomp>
    return [self._serialize(d, many=False) for d in obj]
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow/schema.py", line 503, in _serialize
    value = field_obj.serialize(attr_name, obj, accessor=self.get_attribute)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow/fields.py", line 301, in serialize
    value = self.get_value(obj, attr, accessor=accessor)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow_sqlalchemy/fields.py", line 26, in get_value
    return super(fields.List, self).get_value(obj, attr, accessor=accessor)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow/fields.py", line 229, in get_value
    return accessor_func(obj, check_key, default)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow/schema.py", line 465, in get_attribute
    return get_value(obj, attr, default)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow/utils.py", line 227, in get_value
    return _get_value_for_key(obj, key, default)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow/utils.py", line 241, in _get_value_for_key
    return getattr(obj, key, default)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/orm/attributes.py", line 282, in __get__
    return self.impl.get(instance_state(instance), dict_)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/orm/attributes.py", line 710, in get
    value = self.callable_(state, passive)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/orm/strategies.py", line 725, in _load_for_state
    session, state, primary_key_identity, passive
  File "<string>", line 1, in <lambda>
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/orm/strategies.py", line 867, in _emit_lazyload
    .with_post_criteria(set_default_params)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/ext/baked.py", line 531, in all
    return list(self)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/ext/baked.py", line 434, in __iter__
    return q._execute_and_instances(context)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/orm/query.py", line 3359, in _execute_and_instances
    result = conn.execute(querycontext.statement, self._params)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 988, in execute
    return meth(self, multiparams, params)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/sql/elements.py", line 287, in _execute_on_connection
    return connection._execute_clauseelement(self, multiparams, params)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1107, in _execute_clauseelement
    distilled_params,
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1253, in _execute_context
    e, statement, parameters, cursor, context
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1473, in _handle_dbapi_exception
    util.raise_from_cause(sqlalchemy_exception, exc_info)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 398, in raise_from_cause
    reraise(type(exception), exception, tb=exc_tb, cause=cause)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 152, in reraise
    raise value.with_traceback(tb)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1249, in _execute_context
    cursor, statement, parameters, context
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/engine/default.py", line 552, in do_execute
    cursor.execute(statement, parameters)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/MySQLdb/cursors.py", line 209, in execute
    res = self._query(query)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/MySQLdb/cursors.py", line 315, in _query
    db.query(q)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/MySQLdb/connections.py", line 226, in query
    _mysql.connection.query(self, query)
sqlalchemy.exc.OperationalError: (MySQLdb._exceptions.OperationalError) (1054, "Unknown column 'nsr_table_information.Qty_Apr' in 'field list'")
[SQL: SELECT nsr_table_information.id AS nsr_table_information_id, nsr_table_information.master_product_id AS nsr_table_information_master_product_id, nsr_table_information.segment_id AS nsr_table_information_segment_id, nsr_table_information.paper_machine_id AS nsr_table_information_paper_machine_id, nsr_table_information.last_fy_nsr_value AS nsr_table_information_last_fy_nsr_value, nsr_table_information.`Qty_Apr` AS `nsr_table_information_Qty_Apr`, nsr_table_information.`Qty_May` AS `nsr_table_information_Qty_May`, nsr_table_information.`Qty_Jun` AS `nsr_table_information_Qty_Jun`, nsr_table_information.`Qty_Jul` AS `nsr_table_information_Qty_Jul`, nsr_table_information.`Qty_Aug` AS `nsr_table_information_Qty_Aug`, nsr_table_information.`Qty_Sep` AS `nsr_table_information_Qty_Sep`, nsr_table_information.`Qty_Oct` AS `nsr_table_information_Qty_Oct`, nsr_table_information.`Qty_Nov` AS `nsr_table_information_Qty_Nov`, nsr_table_information.`Qty_Dec` AS `nsr_table_information_Qty_Dec`, nsr_table_information.`Qty_Jan` AS `nsr_table_information_Qty_Jan`, nsr_table_information.`Qty_Feb` AS `nsr_table_information_Qty_Feb`, nsr_table_information.`Qty_Mar` AS `nsr_table_information_Qty_Mar`, nsr_table_information.`Qty_Total` AS `nsr_table_information_Qty_Total`, nsr_table_information.`NSR_Apr` AS `nsr_table_information_NSR_Apr`, nsr_table_information.`NSR_May` AS `nsr_table_information_NSR_May`, nsr_table_information.`NSR_Jun` AS `nsr_table_information_NSR_Jun`, nsr_table_information.`NSR_Jul` AS `nsr_table_information_NSR_Jul`, nsr_table_information.`NSR_Aug` AS `nsr_table_information_NSR_Aug`, nsr_table_information.`NSR_Sep` AS `nsr_table_information_NSR_Sep`, nsr_table_information.`NSR_Oct` AS `nsr_table_information_NSR_Oct`, nsr_table_information.`NSR_Nov` AS `nsr_table_information_NSR_Nov`, nsr_table_information.`NSR_Dec` AS `nsr_table_information_NSR_Dec`, nsr_table_information.`NSR_Jan` AS `nsr_table_information_NSR_Jan`, nsr_table_information.`NSR_Feb` AS `nsr_table_information_NSR_Feb`, nsr_table_information.`NSR_Mar` AS `nsr_table_information_NSR_Mar`, nsr_table_information.`NSR_Total` AS `nsr_table_information_NSR_Total`, nsr_table_information.grand_totals AS nsr_table_information_grand_totals, nsr_table_information.planned_volume_for_last_fy AS nsr_table_information_planned_volume_for_last_fy, nsr_table_information.planned_nsr_for_last_fy AS nsr_table_information_planned_nsr_for_last_fy, nsr_table_information.ytd_actual_volume_for_fy AS nsr_table_information_ytd_actual_volume_for_fy, nsr_table_information.ytd_actual_nsr_for_fy AS nsr_table_information_ytd_actual_nsr_for_fy, nsr_table_information.status AS nsr_table_information_status, nsr_table_information.`createdAt` AS `nsr_table_information_createdAt`, nsr_table_information.`updatedAt` AS `nsr_table_information_updatedAt` 
FROM nsr_table_information 
WHERE %s = nsr_table_information.segment_id]
[parameters: (1,)]
(Background on this error at: http://sqlalche.me/e/e3q8)
[2019-09-11 18:12:11,092] {add_segment_master.py:42} ERROR - Exception occured
Traceback (most recent call last):
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1249, in _execute_context
    cursor, statement, parameters, context
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/engine/default.py", line 552, in do_execute
    cursor.execute(statement, parameters)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/MySQLdb/cursors.py", line 209, in execute
    res = self._query(query)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/MySQLdb/cursors.py", line 315, in _query
    db.query(q)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/MySQLdb/connections.py", line 226, in query
    _mysql.connection.query(self, query)
MySQLdb._exceptions.OperationalError: (1054, "Unknown column 'nsr_table_information.Qty_Apr' in 'field list'")

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/ganesh/my_projects/itc_poc_server/controllers/segment_master/add_segment_master.py", line 35, in get
    data = data_schema.dump(categories)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow/schema.py", line 539, in dump
    result = self._serialize(processed_obj, many=many)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow/schema.py", line 500, in _serialize
    return [self._serialize(d, many=False) for d in obj]
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow/schema.py", line 500, in <listcomp>
    return [self._serialize(d, many=False) for d in obj]
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow/schema.py", line 503, in _serialize
    value = field_obj.serialize(attr_name, obj, accessor=self.get_attribute)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow/fields.py", line 301, in serialize
    value = self.get_value(obj, attr, accessor=accessor)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow_sqlalchemy/fields.py", line 26, in get_value
    return super(fields.List, self).get_value(obj, attr, accessor=accessor)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow/fields.py", line 229, in get_value
    return accessor_func(obj, check_key, default)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow/schema.py", line 465, in get_attribute
    return get_value(obj, attr, default)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow/utils.py", line 227, in get_value
    return _get_value_for_key(obj, key, default)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow/utils.py", line 241, in _get_value_for_key
    return getattr(obj, key, default)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/orm/attributes.py", line 282, in __get__
    return self.impl.get(instance_state(instance), dict_)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/orm/attributes.py", line 710, in get
    value = self.callable_(state, passive)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/orm/strategies.py", line 725, in _load_for_state
    session, state, primary_key_identity, passive
  File "<string>", line 1, in <lambda>
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/orm/strategies.py", line 867, in _emit_lazyload
    .with_post_criteria(set_default_params)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/ext/baked.py", line 531, in all
    return list(self)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/ext/baked.py", line 434, in __iter__
    return q._execute_and_instances(context)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/orm/query.py", line 3359, in _execute_and_instances
    result = conn.execute(querycontext.statement, self._params)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 988, in execute
    return meth(self, multiparams, params)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/sql/elements.py", line 287, in _execute_on_connection
    return connection._execute_clauseelement(self, multiparams, params)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1107, in _execute_clauseelement
    distilled_params,
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1253, in _execute_context
    e, statement, parameters, cursor, context
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1473, in _handle_dbapi_exception
    util.raise_from_cause(sqlalchemy_exception, exc_info)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 398, in raise_from_cause
    reraise(type(exception), exception, tb=exc_tb, cause=cause)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 152, in reraise
    raise value.with_traceback(tb)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1249, in _execute_context
    cursor, statement, parameters, context
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/engine/default.py", line 552, in do_execute
    cursor.execute(statement, parameters)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/MySQLdb/cursors.py", line 209, in execute
    res = self._query(query)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/MySQLdb/cursors.py", line 315, in _query
    db.query(q)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/MySQLdb/connections.py", line 226, in query
    _mysql.connection.query(self, query)
sqlalchemy.exc.OperationalError: (MySQLdb._exceptions.OperationalError) (1054, "Unknown column 'nsr_table_information.Qty_Apr' in 'field list'")
[SQL: SELECT nsr_table_information.id AS nsr_table_information_id, nsr_table_information.master_product_id AS nsr_table_information_master_product_id, nsr_table_information.segment_id AS nsr_table_information_segment_id, nsr_table_information.paper_machine_id AS nsr_table_information_paper_machine_id, nsr_table_information.last_fy_nsr_value AS nsr_table_information_last_fy_nsr_value, nsr_table_information.`Qty_Apr` AS `nsr_table_information_Qty_Apr`, nsr_table_information.`Qty_May` AS `nsr_table_information_Qty_May`, nsr_table_information.`Qty_Jun` AS `nsr_table_information_Qty_Jun`, nsr_table_information.`Qty_Jul` AS `nsr_table_information_Qty_Jul`, nsr_table_information.`Qty_Aug` AS `nsr_table_information_Qty_Aug`, nsr_table_information.`Qty_Sep` AS `nsr_table_information_Qty_Sep`, nsr_table_information.`Qty_Oct` AS `nsr_table_information_Qty_Oct`, nsr_table_information.`Qty_Nov` AS `nsr_table_information_Qty_Nov`, nsr_table_information.`Qty_Dec` AS `nsr_table_information_Qty_Dec`, nsr_table_information.`Qty_Jan` AS `nsr_table_information_Qty_Jan`, nsr_table_information.`Qty_Feb` AS `nsr_table_information_Qty_Feb`, nsr_table_information.`Qty_Mar` AS `nsr_table_information_Qty_Mar`, nsr_table_information.`Qty_Total` AS `nsr_table_information_Qty_Total`, nsr_table_information.`NSR_Apr` AS `nsr_table_information_NSR_Apr`, nsr_table_information.`NSR_May` AS `nsr_table_information_NSR_May`, nsr_table_information.`NSR_Jun` AS `nsr_table_information_NSR_Jun`, nsr_table_information.`NSR_Jul` AS `nsr_table_information_NSR_Jul`, nsr_table_information.`NSR_Aug` AS `nsr_table_information_NSR_Aug`, nsr_table_information.`NSR_Sep` AS `nsr_table_information_NSR_Sep`, nsr_table_information.`NSR_Oct` AS `nsr_table_information_NSR_Oct`, nsr_table_information.`NSR_Nov` AS `nsr_table_information_NSR_Nov`, nsr_table_information.`NSR_Dec` AS `nsr_table_information_NSR_Dec`, nsr_table_information.`NSR_Jan` AS `nsr_table_information_NSR_Jan`, nsr_table_information.`NSR_Feb` AS `nsr_table_information_NSR_Feb`, nsr_table_information.`NSR_Mar` AS `nsr_table_information_NSR_Mar`, nsr_table_information.`NSR_Total` AS `nsr_table_information_NSR_Total`, nsr_table_information.grand_totals AS nsr_table_information_grand_totals, nsr_table_information.planned_volume_for_last_fy AS nsr_table_information_planned_volume_for_last_fy, nsr_table_information.planned_nsr_for_last_fy AS nsr_table_information_planned_nsr_for_last_fy, nsr_table_information.ytd_actual_volume_for_fy AS nsr_table_information_ytd_actual_volume_for_fy, nsr_table_information.ytd_actual_nsr_for_fy AS nsr_table_information_ytd_actual_nsr_for_fy, nsr_table_information.status AS nsr_table_information_status, nsr_table_information.`createdAt` AS `nsr_table_information_createdAt`, nsr_table_information.`updatedAt` AS `nsr_table_information_updatedAt` 
FROM nsr_table_information 
WHERE %s = nsr_table_information.segment_id]
[parameters: (1,)]
(Background on this error at: http://sqlalche.me/e/e3q8)
[2019-09-11 18:13:13,784] {add_segment_master.py:42} ERROR - Exception occured
Traceback (most recent call last):
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1249, in _execute_context
    cursor, statement, parameters, context
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/engine/default.py", line 552, in do_execute
    cursor.execute(statement, parameters)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/MySQLdb/cursors.py", line 209, in execute
    res = self._query(query)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/MySQLdb/cursors.py", line 315, in _query
    db.query(q)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/MySQLdb/connections.py", line 226, in query
    _mysql.connection.query(self, query)
MySQLdb._exceptions.OperationalError: (1054, "Unknown column 'nsr_table_information.Qty_Apr' in 'field list'")

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/ganesh/my_projects/itc_poc_server/controllers/segment_master/add_segment_master.py", line 35, in get
    data = data_schema.dump(categories)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow/schema.py", line 539, in dump
    result = self._serialize(processed_obj, many=many)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow/schema.py", line 500, in _serialize
    return [self._serialize(d, many=False) for d in obj]
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow/schema.py", line 500, in <listcomp>
    return [self._serialize(d, many=False) for d in obj]
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow/schema.py", line 503, in _serialize
    value = field_obj.serialize(attr_name, obj, accessor=self.get_attribute)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow/fields.py", line 301, in serialize
    value = self.get_value(obj, attr, accessor=accessor)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow_sqlalchemy/fields.py", line 26, in get_value
    return super(fields.List, self).get_value(obj, attr, accessor=accessor)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow/fields.py", line 229, in get_value
    return accessor_func(obj, check_key, default)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow/schema.py", line 465, in get_attribute
    return get_value(obj, attr, default)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow/utils.py", line 227, in get_value
    return _get_value_for_key(obj, key, default)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow/utils.py", line 241, in _get_value_for_key
    return getattr(obj, key, default)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/orm/attributes.py", line 282, in __get__
    return self.impl.get(instance_state(instance), dict_)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/orm/attributes.py", line 710, in get
    value = self.callable_(state, passive)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/orm/strategies.py", line 725, in _load_for_state
    session, state, primary_key_identity, passive
  File "<string>", line 1, in <lambda>
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/orm/strategies.py", line 867, in _emit_lazyload
    .with_post_criteria(set_default_params)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/ext/baked.py", line 531, in all
    return list(self)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/ext/baked.py", line 434, in __iter__
    return q._execute_and_instances(context)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/orm/query.py", line 3359, in _execute_and_instances
    result = conn.execute(querycontext.statement, self._params)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 988, in execute
    return meth(self, multiparams, params)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/sql/elements.py", line 287, in _execute_on_connection
    return connection._execute_clauseelement(self, multiparams, params)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1107, in _execute_clauseelement
    distilled_params,
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1253, in _execute_context
    e, statement, parameters, cursor, context
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1473, in _handle_dbapi_exception
    util.raise_from_cause(sqlalchemy_exception, exc_info)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 398, in raise_from_cause
    reraise(type(exception), exception, tb=exc_tb, cause=cause)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 152, in reraise
    raise value.with_traceback(tb)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1249, in _execute_context
    cursor, statement, parameters, context
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/engine/default.py", line 552, in do_execute
    cursor.execute(statement, parameters)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/MySQLdb/cursors.py", line 209, in execute
    res = self._query(query)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/MySQLdb/cursors.py", line 315, in _query
    db.query(q)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/MySQLdb/connections.py", line 226, in query
    _mysql.connection.query(self, query)
sqlalchemy.exc.OperationalError: (MySQLdb._exceptions.OperationalError) (1054, "Unknown column 'nsr_table_information.Qty_Apr' in 'field list'")
[SQL: SELECT nsr_table_information.id AS nsr_table_information_id, nsr_table_information.master_product_id AS nsr_table_information_master_product_id, nsr_table_information.segment_id AS nsr_table_information_segment_id, nsr_table_information.paper_machine_id AS nsr_table_information_paper_machine_id, nsr_table_information.last_fy_nsr_value AS nsr_table_information_last_fy_nsr_value, nsr_table_information.`Qty_Apr` AS `nsr_table_information_Qty_Apr`, nsr_table_information.`Qty_May` AS `nsr_table_information_Qty_May`, nsr_table_information.`Qty_Jun` AS `nsr_table_information_Qty_Jun`, nsr_table_information.`Qty_Jul` AS `nsr_table_information_Qty_Jul`, nsr_table_information.`Qty_Aug` AS `nsr_table_information_Qty_Aug`, nsr_table_information.`Qty_Sep` AS `nsr_table_information_Qty_Sep`, nsr_table_information.`Qty_Oct` AS `nsr_table_information_Qty_Oct`, nsr_table_information.`Qty_Nov` AS `nsr_table_information_Qty_Nov`, nsr_table_information.`Qty_Dec` AS `nsr_table_information_Qty_Dec`, nsr_table_information.`Qty_Jan` AS `nsr_table_information_Qty_Jan`, nsr_table_information.`Qty_Feb` AS `nsr_table_information_Qty_Feb`, nsr_table_information.`Qty_Mar` AS `nsr_table_information_Qty_Mar`, nsr_table_information.`Qty_Total` AS `nsr_table_information_Qty_Total`, nsr_table_information.`NSR_Apr` AS `nsr_table_information_NSR_Apr`, nsr_table_information.`NSR_May` AS `nsr_table_information_NSR_May`, nsr_table_information.`NSR_Jun` AS `nsr_table_information_NSR_Jun`, nsr_table_information.`NSR_Jul` AS `nsr_table_information_NSR_Jul`, nsr_table_information.`NSR_Aug` AS `nsr_table_information_NSR_Aug`, nsr_table_information.`NSR_Sep` AS `nsr_table_information_NSR_Sep`, nsr_table_information.`NSR_Oct` AS `nsr_table_information_NSR_Oct`, nsr_table_information.`NSR_Nov` AS `nsr_table_information_NSR_Nov`, nsr_table_information.`NSR_Dec` AS `nsr_table_information_NSR_Dec`, nsr_table_information.`NSR_Jan` AS `nsr_table_information_NSR_Jan`, nsr_table_information.`NSR_Feb` AS `nsr_table_information_NSR_Feb`, nsr_table_information.`NSR_Mar` AS `nsr_table_information_NSR_Mar`, nsr_table_information.`NSR_Total` AS `nsr_table_information_NSR_Total`, nsr_table_information.grand_totals AS nsr_table_information_grand_totals, nsr_table_information.planned_volume_for_last_fy AS nsr_table_information_planned_volume_for_last_fy, nsr_table_information.planned_nsr_for_last_fy AS nsr_table_information_planned_nsr_for_last_fy, nsr_table_information.ytd_actual_volume_for_fy AS nsr_table_information_ytd_actual_volume_for_fy, nsr_table_information.ytd_actual_nsr_for_fy AS nsr_table_information_ytd_actual_nsr_for_fy, nsr_table_information.status AS nsr_table_information_status, nsr_table_information.`createdAt` AS `nsr_table_information_createdAt`, nsr_table_information.`updatedAt` AS `nsr_table_information_updatedAt` 
FROM nsr_table_information 
WHERE %s = nsr_table_information.segment_id]
[parameters: (1,)]
(Background on this error at: http://sqlalche.me/e/e3q8)
[2019-09-11 18:13:15,100] {add_segment_master.py:42} ERROR - Exception occured
Traceback (most recent call last):
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1249, in _execute_context
    cursor, statement, parameters, context
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/engine/default.py", line 552, in do_execute
    cursor.execute(statement, parameters)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/MySQLdb/cursors.py", line 209, in execute
    res = self._query(query)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/MySQLdb/cursors.py", line 315, in _query
    db.query(q)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/MySQLdb/connections.py", line 226, in query
    _mysql.connection.query(self, query)
MySQLdb._exceptions.OperationalError: (1054, "Unknown column 'nsr_table_information.Qty_Apr' in 'field list'")

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/ganesh/my_projects/itc_poc_server/controllers/segment_master/add_segment_master.py", line 35, in get
    data = data_schema.dump(categories)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow/schema.py", line 539, in dump
    result = self._serialize(processed_obj, many=many)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow/schema.py", line 500, in _serialize
    return [self._serialize(d, many=False) for d in obj]
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow/schema.py", line 500, in <listcomp>
    return [self._serialize(d, many=False) for d in obj]
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow/schema.py", line 503, in _serialize
    value = field_obj.serialize(attr_name, obj, accessor=self.get_attribute)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow/fields.py", line 301, in serialize
    value = self.get_value(obj, attr, accessor=accessor)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow_sqlalchemy/fields.py", line 26, in get_value
    return super(fields.List, self).get_value(obj, attr, accessor=accessor)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow/fields.py", line 229, in get_value
    return accessor_func(obj, check_key, default)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow/schema.py", line 465, in get_attribute
    return get_value(obj, attr, default)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow/utils.py", line 227, in get_value
    return _get_value_for_key(obj, key, default)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow/utils.py", line 241, in _get_value_for_key
    return getattr(obj, key, default)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/orm/attributes.py", line 282, in __get__
    return self.impl.get(instance_state(instance), dict_)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/orm/attributes.py", line 710, in get
    value = self.callable_(state, passive)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/orm/strategies.py", line 725, in _load_for_state
    session, state, primary_key_identity, passive
  File "<string>", line 1, in <lambda>
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/orm/strategies.py", line 867, in _emit_lazyload
    .with_post_criteria(set_default_params)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/ext/baked.py", line 531, in all
    return list(self)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/ext/baked.py", line 434, in __iter__
    return q._execute_and_instances(context)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/orm/query.py", line 3359, in _execute_and_instances
    result = conn.execute(querycontext.statement, self._params)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 988, in execute
    return meth(self, multiparams, params)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/sql/elements.py", line 287, in _execute_on_connection
    return connection._execute_clauseelement(self, multiparams, params)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1107, in _execute_clauseelement
    distilled_params,
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1253, in _execute_context
    e, statement, parameters, cursor, context
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1473, in _handle_dbapi_exception
    util.raise_from_cause(sqlalchemy_exception, exc_info)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 398, in raise_from_cause
    reraise(type(exception), exception, tb=exc_tb, cause=cause)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 152, in reraise
    raise value.with_traceback(tb)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1249, in _execute_context
    cursor, statement, parameters, context
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/engine/default.py", line 552, in do_execute
    cursor.execute(statement, parameters)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/MySQLdb/cursors.py", line 209, in execute
    res = self._query(query)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/MySQLdb/cursors.py", line 315, in _query
    db.query(q)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/MySQLdb/connections.py", line 226, in query
    _mysql.connection.query(self, query)
sqlalchemy.exc.OperationalError: (MySQLdb._exceptions.OperationalError) (1054, "Unknown column 'nsr_table_information.Qty_Apr' in 'field list'")
[SQL: SELECT nsr_table_information.id AS nsr_table_information_id, nsr_table_information.master_product_id AS nsr_table_information_master_product_id, nsr_table_information.segment_id AS nsr_table_information_segment_id, nsr_table_information.paper_machine_id AS nsr_table_information_paper_machine_id, nsr_table_information.last_fy_nsr_value AS nsr_table_information_last_fy_nsr_value, nsr_table_information.`Qty_Apr` AS `nsr_table_information_Qty_Apr`, nsr_table_information.`Qty_May` AS `nsr_table_information_Qty_May`, nsr_table_information.`Qty_Jun` AS `nsr_table_information_Qty_Jun`, nsr_table_information.`Qty_Jul` AS `nsr_table_information_Qty_Jul`, nsr_table_information.`Qty_Aug` AS `nsr_table_information_Qty_Aug`, nsr_table_information.`Qty_Sep` AS `nsr_table_information_Qty_Sep`, nsr_table_information.`Qty_Oct` AS `nsr_table_information_Qty_Oct`, nsr_table_information.`Qty_Nov` AS `nsr_table_information_Qty_Nov`, nsr_table_information.`Qty_Dec` AS `nsr_table_information_Qty_Dec`, nsr_table_information.`Qty_Jan` AS `nsr_table_information_Qty_Jan`, nsr_table_information.`Qty_Feb` AS `nsr_table_information_Qty_Feb`, nsr_table_information.`Qty_Mar` AS `nsr_table_information_Qty_Mar`, nsr_table_information.`Qty_Total` AS `nsr_table_information_Qty_Total`, nsr_table_information.`NSR_Apr` AS `nsr_table_information_NSR_Apr`, nsr_table_information.`NSR_May` AS `nsr_table_information_NSR_May`, nsr_table_information.`NSR_Jun` AS `nsr_table_information_NSR_Jun`, nsr_table_information.`NSR_Jul` AS `nsr_table_information_NSR_Jul`, nsr_table_information.`NSR_Aug` AS `nsr_table_information_NSR_Aug`, nsr_table_information.`NSR_Sep` AS `nsr_table_information_NSR_Sep`, nsr_table_information.`NSR_Oct` AS `nsr_table_information_NSR_Oct`, nsr_table_information.`NSR_Nov` AS `nsr_table_information_NSR_Nov`, nsr_table_information.`NSR_Dec` AS `nsr_table_information_NSR_Dec`, nsr_table_information.`NSR_Jan` AS `nsr_table_information_NSR_Jan`, nsr_table_information.`NSR_Feb` AS `nsr_table_information_NSR_Feb`, nsr_table_information.`NSR_Mar` AS `nsr_table_information_NSR_Mar`, nsr_table_information.`NSR_Total` AS `nsr_table_information_NSR_Total`, nsr_table_information.grand_totals AS nsr_table_information_grand_totals, nsr_table_information.planned_volume_for_last_fy AS nsr_table_information_planned_volume_for_last_fy, nsr_table_information.planned_nsr_for_last_fy AS nsr_table_information_planned_nsr_for_last_fy, nsr_table_information.ytd_actual_volume_for_fy AS nsr_table_information_ytd_actual_volume_for_fy, nsr_table_information.ytd_actual_nsr_for_fy AS nsr_table_information_ytd_actual_nsr_for_fy, nsr_table_information.status AS nsr_table_information_status, nsr_table_information.`createdAt` AS `nsr_table_information_createdAt`, nsr_table_information.`updatedAt` AS `nsr_table_information_updatedAt` 
FROM nsr_table_information 
WHERE %s = nsr_table_information.segment_id]
[parameters: (1,)]
(Background on this error at: http://sqlalche.me/e/e3q8)
[2019-09-11 18:15:11,826] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 18:19:42,748] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 18:21:46,283] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 18:22:44,593] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 18:22:45,453] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 18:29:31,037] {add_segment_master.py:42} ERROR - Exception occured
Traceback (most recent call last):
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1249, in _execute_context
    cursor, statement, parameters, context
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/engine/default.py", line 552, in do_execute
    cursor.execute(statement, parameters)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/MySQLdb/cursors.py", line 209, in execute
    res = self._query(query)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/MySQLdb/cursors.py", line 315, in _query
    db.query(q)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/MySQLdb/connections.py", line 226, in query
    _mysql.connection.query(self, query)
MySQLdb._exceptions.OperationalError: (1054, "Unknown column 'nsr_table_information.Qty_Apr' in 'field list'")

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/ganesh/my_projects/itc_poc_server/controllers/segment_master/add_segment_master.py", line 35, in get
    data = data_schema.dump(categories)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow/schema.py", line 539, in dump
    result = self._serialize(processed_obj, many=many)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow/schema.py", line 500, in _serialize
    return [self._serialize(d, many=False) for d in obj]
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow/schema.py", line 500, in <listcomp>
    return [self._serialize(d, many=False) for d in obj]
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow/schema.py", line 503, in _serialize
    value = field_obj.serialize(attr_name, obj, accessor=self.get_attribute)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow/fields.py", line 301, in serialize
    value = self.get_value(obj, attr, accessor=accessor)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow_sqlalchemy/fields.py", line 26, in get_value
    return super(fields.List, self).get_value(obj, attr, accessor=accessor)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow/fields.py", line 229, in get_value
    return accessor_func(obj, check_key, default)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow/schema.py", line 465, in get_attribute
    return get_value(obj, attr, default)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow/utils.py", line 227, in get_value
    return _get_value_for_key(obj, key, default)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow/utils.py", line 241, in _get_value_for_key
    return getattr(obj, key, default)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/orm/attributes.py", line 282, in __get__
    return self.impl.get(instance_state(instance), dict_)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/orm/attributes.py", line 710, in get
    value = self.callable_(state, passive)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/orm/strategies.py", line 725, in _load_for_state
    session, state, primary_key_identity, passive
  File "<string>", line 1, in <lambda>
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/orm/strategies.py", line 867, in _emit_lazyload
    .with_post_criteria(set_default_params)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/ext/baked.py", line 531, in all
    return list(self)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/ext/baked.py", line 434, in __iter__
    return q._execute_and_instances(context)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/orm/query.py", line 3359, in _execute_and_instances
    result = conn.execute(querycontext.statement, self._params)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 988, in execute
    return meth(self, multiparams, params)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/sql/elements.py", line 287, in _execute_on_connection
    return connection._execute_clauseelement(self, multiparams, params)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1107, in _execute_clauseelement
    distilled_params,
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1253, in _execute_context
    e, statement, parameters, cursor, context
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1473, in _handle_dbapi_exception
    util.raise_from_cause(sqlalchemy_exception, exc_info)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 398, in raise_from_cause
    reraise(type(exception), exception, tb=exc_tb, cause=cause)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 152, in reraise
    raise value.with_traceback(tb)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1249, in _execute_context
    cursor, statement, parameters, context
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/engine/default.py", line 552, in do_execute
    cursor.execute(statement, parameters)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/MySQLdb/cursors.py", line 209, in execute
    res = self._query(query)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/MySQLdb/cursors.py", line 315, in _query
    db.query(q)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/MySQLdb/connections.py", line 226, in query
    _mysql.connection.query(self, query)
sqlalchemy.exc.OperationalError: (MySQLdb._exceptions.OperationalError) (1054, "Unknown column 'nsr_table_information.Qty_Apr' in 'field list'")
[SQL: SELECT nsr_table_information.id AS nsr_table_information_id, nsr_table_information.master_product_id AS nsr_table_information_master_product_id, nsr_table_information.segment_id AS nsr_table_information_segment_id, nsr_table_information.paper_machine_id AS nsr_table_information_paper_machine_id, nsr_table_information.last_fy_nsr_value AS nsr_table_information_last_fy_nsr_value, nsr_table_information.`Qty_Apr` AS `nsr_table_information_Qty_Apr`, nsr_table_information.`Qty_May` AS `nsr_table_information_Qty_May`, nsr_table_information.`Qty_Jun` AS `nsr_table_information_Qty_Jun`, nsr_table_information.`Qty_Jul` AS `nsr_table_information_Qty_Jul`, nsr_table_information.`Qty_Aug` AS `nsr_table_information_Qty_Aug`, nsr_table_information.`Qty_Sep` AS `nsr_table_information_Qty_Sep`, nsr_table_information.`Qty_Oct` AS `nsr_table_information_Qty_Oct`, nsr_table_information.`Qty_Nov` AS `nsr_table_information_Qty_Nov`, nsr_table_information.`Qty_Dec` AS `nsr_table_information_Qty_Dec`, nsr_table_information.`Qty_Jan` AS `nsr_table_information_Qty_Jan`, nsr_table_information.`Qty_Feb` AS `nsr_table_information_Qty_Feb`, nsr_table_information.`Qty_Mar` AS `nsr_table_information_Qty_Mar`, nsr_table_information.`Qty_Total` AS `nsr_table_information_Qty_Total`, nsr_table_information.`NSR_Apr` AS `nsr_table_information_NSR_Apr`, nsr_table_information.`NSR_May` AS `nsr_table_information_NSR_May`, nsr_table_information.`NSR_Jun` AS `nsr_table_information_NSR_Jun`, nsr_table_information.`NSR_Jul` AS `nsr_table_information_NSR_Jul`, nsr_table_information.`NSR_Aug` AS `nsr_table_information_NSR_Aug`, nsr_table_information.`NSR_Sep` AS `nsr_table_information_NSR_Sep`, nsr_table_information.`NSR_Oct` AS `nsr_table_information_NSR_Oct`, nsr_table_information.`NSR_Nov` AS `nsr_table_information_NSR_Nov`, nsr_table_information.`NSR_Dec` AS `nsr_table_information_NSR_Dec`, nsr_table_information.`NSR_Jan` AS `nsr_table_information_NSR_Jan`, nsr_table_information.`NSR_Feb` AS `nsr_table_information_NSR_Feb`, nsr_table_information.`NSR_Mar` AS `nsr_table_information_NSR_Mar`, nsr_table_information.`NSR_Total` AS `nsr_table_information_NSR_Total`, nsr_table_information.`NSR_Value_Apr` AS `nsr_table_information_NSR_Value_Apr`, nsr_table_information.`NSR_Value_May` AS `nsr_table_information_NSR_Value_May`, nsr_table_information.`NSR_Value_Jun` AS `nsr_table_information_NSR_Value_Jun`, nsr_table_information.`NSR_Value_Jul` AS `nsr_table_information_NSR_Value_Jul`, nsr_table_information.`NSR_Value_Aug` AS `nsr_table_information_NSR_Value_Aug`, nsr_table_information.`NSR_Value_Sep` AS `nsr_table_information_NSR_Value_Sep`, nsr_table_information.`NSR_Value_Oct` AS `nsr_table_information_NSR_Value_Oct`, nsr_table_information.`NSR_Value_Nov` AS `nsr_table_information_NSR_Value_Nov`, nsr_table_information.`NSR_Value_Dec` AS `nsr_table_information_NSR_Value_Dec`, nsr_table_information.`NSR_Value_Jan` AS `nsr_table_information_NSR_Value_Jan`, nsr_table_information.`NSR_Value_Feb` AS `nsr_table_information_NSR_Value_Feb`, nsr_table_information.`NSR_Value_Mar` AS `nsr_table_information_NSR_Value_Mar`, nsr_table_information.`NSR_Value_Total` AS `nsr_table_information_NSR_Value_Total`, nsr_table_information.grand_totals AS nsr_table_information_grand_totals, nsr_table_information.planned_volume_for_last_fy AS nsr_table_information_planned_volume_for_last_fy, nsr_table_information.planned_nsr_for_last_fy AS nsr_table_information_planned_nsr_for_last_fy, nsr_table_information.ytd_actual_volume_for_fy AS nsr_table_information_ytd_actual_volume_for_fy, nsr_table_information.ytd_actual_nsr_for_fy AS nsr_table_information_ytd_actual_nsr_for_fy, nsr_table_information.status AS nsr_table_information_status, nsr_table_information.`createdAt` AS `nsr_table_information_createdAt`, nsr_table_information.`updatedAt` AS `nsr_table_information_updatedAt` 
FROM nsr_table_information 
WHERE %s = nsr_table_information.segment_id]
[parameters: (1,)]
(Background on this error at: http://sqlalche.me/e/e3q8)
[2019-09-11 18:29:31,794] {add_segment_master.py:42} ERROR - Exception occured
Traceback (most recent call last):
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1249, in _execute_context
    cursor, statement, parameters, context
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/engine/default.py", line 552, in do_execute
    cursor.execute(statement, parameters)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/MySQLdb/cursors.py", line 209, in execute
    res = self._query(query)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/MySQLdb/cursors.py", line 315, in _query
    db.query(q)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/MySQLdb/connections.py", line 226, in query
    _mysql.connection.query(self, query)
MySQLdb._exceptions.OperationalError: (1054, "Unknown column 'nsr_table_information.Qty_Apr' in 'field list'")

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/ganesh/my_projects/itc_poc_server/controllers/segment_master/add_segment_master.py", line 35, in get
    data = data_schema.dump(categories)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow/schema.py", line 539, in dump
    result = self._serialize(processed_obj, many=many)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow/schema.py", line 500, in _serialize
    return [self._serialize(d, many=False) for d in obj]
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow/schema.py", line 500, in <listcomp>
    return [self._serialize(d, many=False) for d in obj]
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow/schema.py", line 503, in _serialize
    value = field_obj.serialize(attr_name, obj, accessor=self.get_attribute)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow/fields.py", line 301, in serialize
    value = self.get_value(obj, attr, accessor=accessor)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow_sqlalchemy/fields.py", line 26, in get_value
    return super(fields.List, self).get_value(obj, attr, accessor=accessor)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow/fields.py", line 229, in get_value
    return accessor_func(obj, check_key, default)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow/schema.py", line 465, in get_attribute
    return get_value(obj, attr, default)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow/utils.py", line 227, in get_value
    return _get_value_for_key(obj, key, default)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/marshmallow/utils.py", line 241, in _get_value_for_key
    return getattr(obj, key, default)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/orm/attributes.py", line 282, in __get__
    return self.impl.get(instance_state(instance), dict_)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/orm/attributes.py", line 710, in get
    value = self.callable_(state, passive)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/orm/strategies.py", line 725, in _load_for_state
    session, state, primary_key_identity, passive
  File "<string>", line 1, in <lambda>
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/orm/strategies.py", line 867, in _emit_lazyload
    .with_post_criteria(set_default_params)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/ext/baked.py", line 531, in all
    return list(self)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/ext/baked.py", line 434, in __iter__
    return q._execute_and_instances(context)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/orm/query.py", line 3359, in _execute_and_instances
    result = conn.execute(querycontext.statement, self._params)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 988, in execute
    return meth(self, multiparams, params)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/sql/elements.py", line 287, in _execute_on_connection
    return connection._execute_clauseelement(self, multiparams, params)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1107, in _execute_clauseelement
    distilled_params,
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1253, in _execute_context
    e, statement, parameters, cursor, context
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1473, in _handle_dbapi_exception
    util.raise_from_cause(sqlalchemy_exception, exc_info)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 398, in raise_from_cause
    reraise(type(exception), exception, tb=exc_tb, cause=cause)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/util/compat.py", line 152, in reraise
    raise value.with_traceback(tb)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/engine/base.py", line 1249, in _execute_context
    cursor, statement, parameters, context
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/sqlalchemy/engine/default.py", line 552, in do_execute
    cursor.execute(statement, parameters)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/MySQLdb/cursors.py", line 209, in execute
    res = self._query(query)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/MySQLdb/cursors.py", line 315, in _query
    db.query(q)
  File "/home/ganesh/my_projects/itc_venv/lib/python3.7/site-packages/MySQLdb/connections.py", line 226, in query
    _mysql.connection.query(self, query)
sqlalchemy.exc.OperationalError: (MySQLdb._exceptions.OperationalError) (1054, "Unknown column 'nsr_table_information.Qty_Apr' in 'field list'")
[SQL: SELECT nsr_table_information.id AS nsr_table_information_id, nsr_table_information.master_product_id AS nsr_table_information_master_product_id, nsr_table_information.segment_id AS nsr_table_information_segment_id, nsr_table_information.paper_machine_id AS nsr_table_information_paper_machine_id, nsr_table_information.last_fy_nsr_value AS nsr_table_information_last_fy_nsr_value, nsr_table_information.`Qty_Apr` AS `nsr_table_information_Qty_Apr`, nsr_table_information.`Qty_May` AS `nsr_table_information_Qty_May`, nsr_table_information.`Qty_Jun` AS `nsr_table_information_Qty_Jun`, nsr_table_information.`Qty_Jul` AS `nsr_table_information_Qty_Jul`, nsr_table_information.`Qty_Aug` AS `nsr_table_information_Qty_Aug`, nsr_table_information.`Qty_Sep` AS `nsr_table_information_Qty_Sep`, nsr_table_information.`Qty_Oct` AS `nsr_table_information_Qty_Oct`, nsr_table_information.`Qty_Nov` AS `nsr_table_information_Qty_Nov`, nsr_table_information.`Qty_Dec` AS `nsr_table_information_Qty_Dec`, nsr_table_information.`Qty_Jan` AS `nsr_table_information_Qty_Jan`, nsr_table_information.`Qty_Feb` AS `nsr_table_information_Qty_Feb`, nsr_table_information.`Qty_Mar` AS `nsr_table_information_Qty_Mar`, nsr_table_information.`Qty_Total` AS `nsr_table_information_Qty_Total`, nsr_table_information.`NSR_Apr` AS `nsr_table_information_NSR_Apr`, nsr_table_information.`NSR_May` AS `nsr_table_information_NSR_May`, nsr_table_information.`NSR_Jun` AS `nsr_table_information_NSR_Jun`, nsr_table_information.`NSR_Jul` AS `nsr_table_information_NSR_Jul`, nsr_table_information.`NSR_Aug` AS `nsr_table_information_NSR_Aug`, nsr_table_information.`NSR_Sep` AS `nsr_table_information_NSR_Sep`, nsr_table_information.`NSR_Oct` AS `nsr_table_information_NSR_Oct`, nsr_table_information.`NSR_Nov` AS `nsr_table_information_NSR_Nov`, nsr_table_information.`NSR_Dec` AS `nsr_table_information_NSR_Dec`, nsr_table_information.`NSR_Jan` AS `nsr_table_information_NSR_Jan`, nsr_table_information.`NSR_Feb` AS `nsr_table_information_NSR_Feb`, nsr_table_information.`NSR_Mar` AS `nsr_table_information_NSR_Mar`, nsr_table_information.`NSR_Total` AS `nsr_table_information_NSR_Total`, nsr_table_information.`NSR_Value_Apr` AS `nsr_table_information_NSR_Value_Apr`, nsr_table_information.`NSR_Value_May` AS `nsr_table_information_NSR_Value_May`, nsr_table_information.`NSR_Value_Jun` AS `nsr_table_information_NSR_Value_Jun`, nsr_table_information.`NSR_Value_Jul` AS `nsr_table_information_NSR_Value_Jul`, nsr_table_information.`NSR_Value_Aug` AS `nsr_table_information_NSR_Value_Aug`, nsr_table_information.`NSR_Value_Sep` AS `nsr_table_information_NSR_Value_Sep`, nsr_table_information.`NSR_Value_Oct` AS `nsr_table_information_NSR_Value_Oct`, nsr_table_information.`NSR_Value_Nov` AS `nsr_table_information_NSR_Value_Nov`, nsr_table_information.`NSR_Value_Dec` AS `nsr_table_information_NSR_Value_Dec`, nsr_table_information.`NSR_Value_Jan` AS `nsr_table_information_NSR_Value_Jan`, nsr_table_information.`NSR_Value_Feb` AS `nsr_table_information_NSR_Value_Feb`, nsr_table_information.`NSR_Value_Mar` AS `nsr_table_information_NSR_Value_Mar`, nsr_table_information.`NSR_Value_Total` AS `nsr_table_information_NSR_Value_Total`, nsr_table_information.grand_totals AS nsr_table_information_grand_totals, nsr_table_information.planned_volume_for_last_fy AS nsr_table_information_planned_volume_for_last_fy, nsr_table_information.planned_nsr_for_last_fy AS nsr_table_information_planned_nsr_for_last_fy, nsr_table_information.ytd_actual_volume_for_fy AS nsr_table_information_ytd_actual_volume_for_fy, nsr_table_information.ytd_actual_nsr_for_fy AS nsr_table_information_ytd_actual_nsr_for_fy, nsr_table_information.status AS nsr_table_information_status, nsr_table_information.`createdAt` AS `nsr_table_information_createdAt`, nsr_table_information.`updatedAt` AS `nsr_table_information_updatedAt` 
FROM nsr_table_information 
WHERE %s = nsr_table_information.segment_id]
[parameters: (1,)]
(Background on this error at: http://sqlalche.me/e/e3q8)
[2019-09-11 18:40:22,838] {add_segment_master.py:39} INFO - no data is available on Segment Master
[2019-09-11 18:43:05,611] {add_segment_master.py:39} INFO - no data is available on Segment Master
[2019-09-11 18:43:16,028] {add_segment_master.py:64} INFO - segment master Details posted succesfully
[2019-09-11 18:43:16,458] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 18:43:22,019] {add_segment_master.py:64} INFO - segment master Details posted succesfully
[2019-09-11 18:43:22,584] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 18:43:27,099] {add_segment_master.py:64} INFO - segment master Details posted succesfully
[2019-09-11 18:43:37,404] {add_segment_master.py:68} INFO - Segment Masters code or name already exists
[2019-09-11 18:43:38,336] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 18:43:55,006] {add_segment_master.py:68} INFO - Segment Masters code or name already exists
[2019-09-11 18:44:00,839] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 18:44:09,233] {add_segment_master.py:64} INFO - segment master Details posted succesfully
[2019-09-11 18:44:10,020] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 18:48:39,311] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 18:48:42,039] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 18:56:55,774] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 18:58:25,712] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 18:59:45,413] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 19:04:13,483] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 19:09:56,757] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 19:10:24,178] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 19:10:31,155] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 19:11:06,144] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 19:11:07,774] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 19:12:08,725] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 19:12:09,165] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 19:12:29,349] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 19:12:31,335] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 19:13:21,345] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 19:13:22,435] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 19:13:45,509] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 19:17:35,021] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 19:19:42,948] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 19:19:43,715] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 19:19:56,804] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 19:22:36,215] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 19:29:12,027] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 19:31:04,932] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 19:32:35,408] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 19:41:50,875] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 19:47:11,845] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 19:47:13,091] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 19:50:08,241] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 19:50:09,770] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 19:53:23,023] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 19:53:25,184] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 19:59:00,387] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 19:59:07,543] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 20:00:00,434] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 20:00:00,992] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 20:02:01,635] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 20:02:01,735] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 20:02:50,373] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 20:02:51,558] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 20:04:48,508] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 20:04:48,517] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 20:12:36,249] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 20:13:21,996] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 20:14:55,115] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 20:22:38,128] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-11 20:33:37,307] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-12 08:17:39,389] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-12 08:22:37,163] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-12 08:22:40,767] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-12 08:22:45,869] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-12 08:23:46,942] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-12 08:23:48,694] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-12 08:23:51,324] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-12 08:24:09,535] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-12 08:24:10,690] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-12 08:25:55,590] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-12 08:25:56,710] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-12 08:28:16,864] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-12 08:46:15,728] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-12 08:46:18,319] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-12 08:47:20,396] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-12 08:52:38,339] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-12 08:53:27,337] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-12 08:54:50,102] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-12 08:56:06,421] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-12 08:56:35,636] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-12 08:59:00,696] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-12 09:04:44,488] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-12 09:07:31,931] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-12 09:07:32,645] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-12 09:11:18,059] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-12 09:11:21,670] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-12 09:15:12,136] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-12 09:17:21,331] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-12 09:23:59,156] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-12 09:23:59,157] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-12 09:24:54,736] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-12 09:24:56,574] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-12 09:25:37,533] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-17 09:47:15,442] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-17 15:55:12,222] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-17 16:04:37,824] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-17 16:05:39,663] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-17 16:06:00,182] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-17 16:30:38,402] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-17 19:09:57,916] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-18 10:10:35,599] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-18 10:11:19,844] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-18 10:18:31,576] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-18 10:20:36,422] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-18 11:46:15,702] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-18 11:49:41,024] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-18 11:49:42,132] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-18 11:50:43,808] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-18 11:51:08,245] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-18 11:53:25,248] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-18 11:53:35,678] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-18 11:56:36,211] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-18 11:56:44,508] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-18 11:57:29,736] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-18 12:03:12,882] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-18 12:03:14,387] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-18 12:04:06,328] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-18 12:04:52,657] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-18 12:08:30,638] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-18 12:08:43,363] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-18 12:10:20,914] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-18 12:13:11,989] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-18 12:13:37,063] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-18 12:16:26,151] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-18 12:19:35,020] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-18 12:56:20,289] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-18 12:59:00,734] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-18 13:01:40,900] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-18 13:03:19,653] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-18 13:05:13,609] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-18 13:06:59,722] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-18 13:08:48,391] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-18 13:11:46,605] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-18 13:12:40,489] {add_segment_master.py:36} INFO - getting data of segment master details is success
[2019-09-25 10:53:28,474] {add_segment_master.py:36} INFO - getting data of segment master details is success
